tests/plenary/job_spec.lua

```diff
@@ -95,9 +95,9 @@ describe("Job", function()
     it("should return last line when there is ending newline", function()
       local results = {}
       local job = Job:new {
-        command = "echo",
+        command = "printf",
 
-        args = { "-e", "test1\ntest2" },
```

Comment:
`-e` option doesn't work in POSIX mode. You can try:
```
$ bash --posix -O xpg_echo -c "echo -e 'A\nB'"
-e A
B
```
Switched to `printf` and it seems to be working fine in both macos and ubuntu.